students/15th/kimbyulyi/js/login.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+const userId = document.querySelector(".user__id");
+const userPw = document.querySelector(".user__pw");
+const loginBtn = document.querySelector(".login__btn");
+const login__form = document.querySelector(".login__form");
+
+const checkUserLogin = (event) => {
+  const emailCheck = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+  const userIdValue = userId.value;
+  const userPwValue = userPw.value;
+  const errorMsg = document.querySelector(".login__error");
+  if (
+    emailCheck.test(String(userIdValue).toLowerCase()) === false ||
+    userPwValue.length < 5
+  ) {
+    errorMsg.innerText = "계정을 다시 확인해주세요";
+  } else {
+    errorMsg.innerText = "로그인 성공";
+  }
```

Comment:
if, else 문 말고 삼항 연산자로 이렇게도 해볼 수 있습니다.
```suggestion
  const isValid = emailCheck.test(String(userIdValue).toLowerCase()) && userPwValue.length >= 5
  errorMsg.innerText = isValid ? "로그인 성공" : "계정을 다시 확인해주세요"
```

Reply:
넵! 참고하겠습니다